protocol/x/clob/keeper/msg_server_create_clob_pair.go

```diff
@@ -2,15 +2,42 @@ package keeper
 
 import (
 	"context"
-	"errors"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 	"github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
 )
 
+// CreateClobPair handles `MsgCreateClobPair`.
 func (k msgServer) CreateClobPair(
```

Comment:
Makes sense, added an error.

For my understanding, why does [PerpetualIdToClobPairId](https://github.com/dydxprotocol/v4-chain/blob/6dc9a4aeea4c54fab71aa60666fe061fcde7d47f/protocol/x/clob/keeper/keeper.go#L35) map a perpetual id to multiple clob pairs? Is this tech debt? If so should we add TODO to refactor?

Reply:
I think because we eventually plan on supporting multiple clob pairs per perpetual. I agree it would make sense to have it map to a single value instead of a slice of clobPairIds. Not super opinionated on refactoring if we may eventually revert the refactor though